### ISSUE TITLE
Setting TLS 1.3 as minumum TLS version yields a penalty for not supporting TLS 1.2

### ISSUE THREAD

**Issue by rmjansen** [ssllabs/ssllabs-scan#930]
Scanning a site with TLS 1.3 as the minimum required version results in an A. Lowering the minimum required TLS version to 1.2 (with no other changes applied) results in an A+ for the same site. This suggests that the penalty for using outdated TLS versions and not supporting TLS 1.2 is also applied to sites that use TLS 1.3 but not support TLS 1.2.

**Comment by ArchangeGabriel** [comment]
Duplicate: #910, #863, #853, #815, #786, #711.
TL;DR: since FALLBACK can’t be tested, you can’t get A+. That’s silly, but the devs never acknowledged this.

**Comment by lilyanatia** [comment]
> since FALLBACK can’t be tested, you can’t get A+. That’s silly, but the devs never acknowledged this.

it's especially silly since:

1. downgrades aren't possible if the server only supports one TLS version.
2. TLS 1.3 includes it's own downgrade prevention mechanism.

**Comment by mulder999** [comment]
It's surprising that this issue still hasn't been resolved in 2024. All major browsers have supported [TLS 1.3](https://caniuse.com/?search=tls%201.3) since 2020 or earlier.

**Comment by kovmir** [comment]
Still not resolved in 2025.

**Comment by siepkes** [comment]
> Still not resolved in 2025.

I agree this is kinda silly. SSL Labs has become well known to everyone, including non-technical people (for example auditors). Congratulations to Qualys on that, that's obviously quite an achievement. However, with that also comes a degree of responsibility. With an issue such as this Qualys is actually starting to harm the advancement of using more secure settings for TLS deployments.
